lnbits/wallets/phoenixd.py

```diff
@@ -87,17 +87,27 @@ async def create_invoice(
         unhashed_description: Optional[bytes] = None,
         **kwargs,
     ) -> InvoiceResponse:
-        if description_hash or unhashed_description:
-            raise UnsupportedError("description_hash")
 
         try:
             msats_amount = amount
             data: Dict = {
                 "amountSat": f"{msats_amount}",
-                "description": memo,
                 "externalId": "",
             }
 
+            # Either 'description' (string) or 'descriptionHash' must be supplied
+            # PhoenixD description limited to 128 characters
+            if description_hash:
+                data["descriptionHash"] = description_hash.hex()
+            else:
+                desc = memo
+                if desc is None and unhashed_description:
+                    desc = unhashed_description.decode()
+                desc = "" if desc is None else desc
+                if len(desc) > 128:
+                    desc = hashlib.sha256(desc.encode("utf-8")).hexdigest()
+                data["description"] = desc
```

Comment:
shouldn't it just set the descriptionHash here, not description? if its hasing?

Reply:
Modified to set descriptionHash if its getting hashed here